Source/WebCore/platform/MIMETypeRegistry.cpp

```diff
@@ -832,4 +833,18 @@ Vector<String> MIMETypeRegistry::allowedFileExtensions(const Vector<String>& mim
     return allowedFileExtensions;
 }
 
+bool MIMETypeRegistry::isJPEGMIMEType(const String& mimeType)
+{
+#if USE(CG)
+    auto destinationUTI = utiFromImageBufferMIMEType(mimeType);
+    if (!destinationUTI)
+        return false;
+    return CFEqual(destinationUTI.get(), jpegUTI());
+#else
+    return mimeType.substring(sizeof "image") == "jpeg"_s;
```

Comment:
It took me a while to realize that `sizeof "image"` will count _also_ the terminating nul character, and therefore the substring will be what follows after `"image/"`. Wouldn't it be better to actually check `mimeType == "image/jpeg"_s`? It's not like there are other MIME types for JPEG images.

Reply:
Fixed.